nomos-services/blend/src/instance.rs

```diff
@@ -225,11 +225,11 @@ where
             }),
             Self::Broadcast(mode) => Ok(Self::Broadcast(mode)),
             Self::BroadcastAfterCore { mode, prev } => {
-                prev.shutdown().await;
+                prev.wait_until_stopped_or_kill().await;
```

Comment:
This should also never happen, since the first time we call this function we move to `Broadcast`. At the very least a warning line should be printed, but maybe even a panic?